packages/picasso-forms/src/Form/story/index.jsx

```diff
@@ -222,3 +222,7 @@ however, you may need custom validators for more complex types of fields.
     title: 'Field requirements',
     description: 'Showcase how to display field requirements.'
   }) // picasso-skip-visuals
+  .addExample('Form/story/Status.example.tsx', {
+    title: 'Status',
```

Comment:
```suggestion
    title: 'Form Level Status Configuration',
```
because just `Status` might not be very clear